.github/triage_replies.md

```diff
@@ -3,37 +3,81 @@
 
-## Visit our mailing list
+
+
+## PRs
+
+### Visit our mailing list
 - Hello, your question seems like a good one to ask on our mailing list at https://groups.google.com/g/awx-project. You can also join #ansible-awx on https://libera.chat/ and ask your question there.
 
-## Create an issue
-- Hello, thanks for reaching out on list. We think this merits an issue on our Github, https://github.com/ansible/awx/issues. If you could open an issue up on Github it will get tagged and integrated into our planning and workflow. All future work will be tracked there.
+### Denied Submission
 
-## Create a Pull Request
-- Hello, we think your idea is good, please consider contributing a PR for this, following our contributing guidelines: https://github.com/ansible/awx/blob/devel/CONTRIBUTING.md
+- Hi!\
+\
+Thanks very much for your submission to AWX. It means a lot to us that you have taken time to contribute.\
+\
+At this time we do not want to merge this PR. Our reasons for this are:\
+\
+(A) INSERT ITEM HERE\
+\
+Please know that we are always up for discussion but this project is very active. Because of this, we're unlikely to see comments made on closed PRs and we lock them after some time. If you or anyone else has any further questions, please let us know by using any of the communication methods listed in the page below:\
+\
+https://github.com/ansible/awx/#get-involved\
+\
+In the future, sometimes starting a discussion on the development list prior to implementing a feature can make getting things included a little easier, but it is not always necessary.\
+\
+Thank you once again for this and your interest in AWX!
+
+
+### No Progress
+- Hi!\
+\
+Thank you very much for your submission to AWX. It means a lot to us that you have taken time to contribute.\
+\
+On this PR changes were requested but it has been some time since then. We think this PR has merit but without the requested changes we are unable to merge it. At this time we are closing you PR. If you get time to address the changes you are welcome to open another PR or we can reopen this PR upon request if you contact us by using any of the communication methods listed in the page below:\
+\
+https://github.com/ansible/awx/#get-involved\
+\
+Thank you once again for this and your interest in AWX!
+
+
+
+
+## Common
 
-## Give us more info
+### Give us more info
 - Hello, we'd love to help but we need a little more information about the problem you're having. Screenshots, log outputs, or any reproducers would be very helpful. 
 
-## Receptor
+### Code of Conduct
+- Hello. Please keep in mind that Ansible adheres to a Code of Conduct in its community spaces. The spirit of the code of conduct is to be kind, and this is your friendly reminder to be so. Please see the full code of conduct here if you have questions: https://docs.ansible.com/ansible/latest/community/code_of_conduct.html 
```

Comment:
```suggestion
- Hello. Please keep in mind that Ansible adheres to a Code of Conduct in its community spaces. The spirit of the code of conduct is to be kind. Please see the full code of conduct here if you have questions: https://docs.ansible.com/ansible/latest/community/code_of_conduct.html 
```

Reply:
I think was intended to be a warning for CoC. @rebeccahhh, is that correct?

Reply:
this is indeed a warning for CoC violations.

Reply:
Resolving since this is a CoC warning.